docs/schema-language.md

```diff
@@ -23,15 +23,18 @@ starting with 0), or manually.
 Each enumeration is defined as follows:
 
 ```cpp
-enum <Name> : <type> {
+enum <Name> : <type> [ : bits ] {
     <value name> [= value],
     ...
 }
 ```
 
+flatdata will auto-generate names for all missing values as `UNKNOWN_VALUE_X` or `UNKNOWN_VALUE_MINUS_X`.
```

Comment:
I have to admit, I don't get this. What is `X`? What happens when bits are not not specified?

Reply:
X is the value, e.g. UNKNOWN_VALUE_1 if 1 hasn't been specified. I will clarify a bit.

If bits are not specified than the full range is assumed, e.g. 8 bits => 256 possible values.
And since all of those could appear in the file (e.g. malicious/corrupted input, etc) they need to be mentioned (otherwise e.g. the transmute done in reading the enum could result in UB)

Reply:
In the Rust backend this will e.g. generate undocumented values that essentially force the user to add a catch-all to matches (unless the enum is exhausting all the available bits anyway).

I was also considering to an alternative solution: Add checks to the read methods (e.g. replace transmute by a giant match statement), but that seemed to violate the 0-overhead paradigm flatdata has (e.g. the user might not even care for broken enum values for many usecases, e.g just filtering for specific values, etc).